types/custom-functions-runtime/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6", 
+            "dom"
```

Comment:
An FYI - we need dom here because it allows us to use console.log in our tests. Thanks.